src/Microsoft.ML.CodeGenerator/Templates/Console/ModelBuilder.cs

```diff
@@ -20,7 +20,7 @@ namespace Microsoft.ML.CodeGenerator.Templates.Console
     /// Class to produce the template output
     /// </summary>
     [global::System.CodeDom.Compiler.GeneratedCodeAttribute("Microsoft.VisualStudio.TextTemplating", "16.0.0.0")]
-    public partial class ModelBuilder : ModelBuilderBase
+    internal partial class ModelBuilder : ModelBuilderBase
```

Comment:
Could you change the corresponding tt file instead. ( You should never change .cs file under Templates since it's all auto-generated by t4 template

Reply:
It is changed. It has

```
visibility="internal"
```